seed_data/bonfires.json

```diff
@@ -725,8 +725,9 @@
     "name": "Symmetric Difference",
     "difficulty": "4.02",
     "description": [
-      "Create a function that takes two or more arrays and returns an array of the symmetric difference of the provided arrays.",
-      "The mathematical term symmetric difference refers to the elements in two sets that are in either the first or second set, but not in both."
+      "Create a function that takes a variable number of arrays as arguments and returns an array of the <a href=\"http://en.wikipedia.org/wiki/Symmetric_difference\">symmetric difference</a> of the provided arrays. Note: symmetric difference is defined not just for any arrays but specifically for sets.",
+      "What does it mean for a binary operation like \"symmetric difference\" to be applied to a variable number of arguments? You can think of it like this: first, apply it to the first two arguments. Remember the result. Apply it again to that result and the third argument. Remember the result. Apply it again... Do that until you're done.
```

Comment:
A bit too hand-holdy for a level 4 bonfire. What you are talking about is iteration, so just say it imo. "Iterate through the arguments, calculating the symmetric difference between the last result and the next element in &lt;arguments&gt;"

Also bear in mind this as been raised in difficulty from a level 2 and a level 4. We'll be adding in a lot of challenges at level 1 and 2 to reinforce array methods.